lib/season_statistics.rb

```diff
@@ -0,0 +1,13 @@
+require_relative "./stat_tracker"
```

Comment:
@Arique1104 I went through and singled out the code that you wrote and added it to this file so you can continue with it